dimod/compatibility23.py

```diff
@@ -35,3 +43,25 @@ def getargspec(f):
         # FullArgSpec(args, varargs, varkw, defaults, kwonlyargs, kwonlydefaults, annotations)
 
         return _ArgSpec(argspec.args, argspec.varargs, argspec.varkw, argspec.defaults)
+
+    # Based on an answer https://stackoverflow.com/a/34757114/8766655
+    # by kindall https://stackoverflow.com/users/416467/kindall
+    class SortKey(object):
```

Comment:
Oh, and since you're defining `SortKey` only in Python3, no need to inherit from `object` (that's the default).